Doc add tenant-federation in authentication-and-authorization (#1689)
grafana/mimir: docs/sources/operators-guide/securing/authentication-and-authorization.md

# Grafana Mimir authentication and authorization

Grafana Mimir is a multi-tenant system where tenants can query metrics and alerts that include their tenant ID.  

> The query takes the tenant ID from the `X-Scope-OrgID` parameter that exists in the HTTP header of each request (e.g. `X-Scope-OrgID: tenant-id`).  

Reviewer: ```suggestion
The query takes the tenant ID from the `X-Scope-OrgID` parameter that exists in the HTTP header of each request, for example `X-Scope-OrgID: tenant-id`.
```
Adhering to style to spell out abbreviations or shortenings.

> Queries can also be federated across multiple tenants when you enabled `tenant-federation`, refer to [Configuration Parameters]({{< relref "../reference-configuration-parameters/index.md#configuration-parameters" >}}). The tenant IDs involved need to be specified separated by a `|` character in the 'X-Scope-OrgID' header. 

Reviewer 2: Could you add an example of what supplying mutiple tenant IDs looks like? E.g. `tenant-1|tenant-2|tenant-3`

> Queries can also be federated across multiple tenants when `tenant-federation` is enabled, refer to [Configuration Parameters]({{< relref "../reference-configuration-parameters/index.md#configuration-parameters" >}}). The tenant IDs involved need to be specified separated by a `|` character in the 'X-Scope-OrgID' header (e.g. `X-Scope-OrgID: tenant-1|tenant-2|tenant-3`).

Reviewer 2: ```suggestion
Queries can also be federated across multiple tenants when `tenant-federation` is enabled, refer to [Configuration Parameters]({{< relref "../configuring/reference-configuration-parameters/index.md#configuration-parameters" >}}). The tenant IDs involved need to be specified separated by a `|` character in the 'X-Scope-OrgID' header (e.g. `X-Scope-OrgID: tenant-1|tenant-2|tenant-3`).
```

Author: Thank you, I've amend the commit.

> Queries can also be federated across multiple tenants when `tenant-federation` is enabled, refer to [Configuration Parameters]({{< relref "../configuring/reference-configuration-parameters/index.md#configuration-parameters" >}}). The tenant IDs involved need to be specified separated by a `|` character in the 'X-Scope-OrgID' header (e.g. `X-Scope-OrgID: tenant-1|tenant-2|tenant-3`).

Reviewer 3: [nit] I would be more direct about how to enable tenant federation.

```suggestion
Queries can also be federated across multiple tenants when tenant federation is enabled setting `-tenant-federation.enabled=true`.
The tenant IDs involved need to be specified separated by a `|` character in the 'X-Scope-OrgID' header (e.g. `X-Scope-OrgID: tenant-1|tenant-2|tenant-3`).
```

Reviewer: > [nit] I would be more direct about how to enable tenant federation.

```suggestion
You can federate queries across multiple tenants by using `true` in `-tenant-federation.enabled=true`. When you specify tenant IDs, separate them with a pipe (`|`) character in the 'X-Scope-OrgID' header, as in the example `X-Scope-OrgID: tenant-1|tenant-2|tenant-3`.
```
